query-engine/js-connectors/src/proxy.rs

```diff
@@ -106,37 +168,143 @@ pub struct Query {
     pub args: Vec<serde_json::Value>,
 }
 
+fn js_planetscale_value_to_quaint(json_value: serde_json::Value, column_type: ColumnType) -> QuaintValue<'static> {
+    // Note(jkomyno): I suggest we avoid so many different panic messages.
+    // They bloat the compiled code, making it heavier. I suggest we consider the type information
+    // received from the JS connector safe, and eventually just let the code panic without a custom message here.
+    // We can later infer which column type is not properly supported from the panic message's line and column anyway.
```

Comment:
```suggestion
```

Reply:
Heh, I wouldn't delete this, but keep it around, according to [this rationale](https://github.com/prisma/prisma-engines/pull/4080/files#r1273358343).

Reply:
Disagree and requesting changes.

Reply:
I see where you coming from, and now that I feel I understood more of your reasons, I agree with you.